app/src/shared/components/Tile2/index.jsx

```diff
@@ -0,0 +1,293 @@
+// @flow
+
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { capital } from 'case'
+import moment from 'moment'
+import { Translate } from 'react-redux-i18n'
+import { DataUnionBadge, IconBadge, DeployingBadge } from './Badge'
+import ImageContainer, { Image } from './ImageContainer'
+import Menu from './Menu'
+import Summary from './Summary'
+import Label from './Label'
+import { RunStates } from '$editor/canvas/state'
+import CanvasPreview from '$editor/canvas/components/Preview'
+import DashboardPreview from '$editor/dashboard/components/Preview'
+import Link from '$shared/components/Link'
+import { formatPath } from '$shared/utils/url'
+import { isPaidProduct } from '$mp/utils/product'
+import { timeUnits } from '$shared/utils/constants'
+import PaymentRate from '$mp/components/PaymentRate'
+import links from '$app/src/links'
+
+const Tile = styled.div`
+    position: relative;
+
+    ${Menu} {
+        opacity: 0;
+        pointer-events: none;
+        transition-property: visibility, opacity;
+        transition: 200ms;
+        visibility: hidden;
+    }
+
+    ${Menu}.show,
+    :hover ${Menu},
+    :focus ${Menu} {
+        opacity: 1;
+        pointer-events: auto;
+        visibility: visible;
+    }
+
+    ${({ suppressHover }) => !suppressHover && css`
+        ${Image} {
+            filter: brightness(100%);
+            transition: 240ms ease-out filter;
+        }
+
+        ${Menu}.show + a ${Image},
+        :hover ${Image} {
+            filter: brightness(70%);
+            transition-duration: 40ms;
+        }
+    `}
+`
+
+type ImageTileProps = {
+    alt?: ?string,
+    height?: any,
+    showDataUnionBadge?: boolean,
+    src?: ?string,
+}
+
+const ImageTile = ({
+    alt,
+    height,
+    showDataUnionBadge,
+    src,
+    ...props
+}: ImageTileProps) => (
+    <Tile {...props} suppressHover>
+        <ImageContainer
+            alt={alt || ''}
+            height={height}
+            src={src || ''}
+        >
+            {!!showDataUnionBadge && (
+                <DataUnionBadge top left />
+            )}
+        </ImageContainer>
+    </Tile>
+)
+
+type CanvasTileProps = {
+    canvas: any,
+    onMenuToggle?: (boolean) => any,
+    actions: any,
+}
+
+export const updatedAt = ({ updated, created }: any): string => `
```

Comment:
@tumppi @mondoreale I think there's some overlap here with `ago` and `updatedAt`?
https://github.com/streamr-dev/streamr-platform/pull/910/files

Reply:
Good spot. We should use `ago` instead of `moment().fromNow()` 👍